src/main/java/seedu/eduke8/storage/TopicsStorage.java

```diff
@@ -97,7 +92,7 @@ private Topic parseToTopicObject(JSONObject topic) throws Eduke8Exception {
     private Question parseToQuestionObject(JSONObject question) throws Eduke8Exception {
         currentQuestionDescription = (String) question.get(KEY_DESCRIPTION);
 
-        checkIfBlank(currentQuestionDescription);
+        checkIfBlankOrDuplicate(currentQuestionDescription, questionDescriptions);
```

Comment:
As discussed, this will be important to check whether an error needs to thrown or not.